test/lint/lint-python-mutable-default-parameters.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Detect when a mutable list or dict is used as a default parameter value in a Python function.
+"""
+
+import subprocess
+import sys
+
+
+def main():
+    command = "git grep -E".split(" ")
+    command.append(r"^\s*def [a-zA-Z0-9_]+\(.*=\s*(\[|\{)")
+    command.extend("-- *.py".split(" "))
+    output = subprocess.run(command, stdout=subprocess.PIPE, universal_newlines=True)
+    output = output.stdout
```

Comment:
Would prefer to just use `output.stdout` in following code instead of this assignment.

Reply:
Done